src/main/webapp/index.jsp

```diff
@@ -77,6 +77,16 @@
     <h1>Welcome!</h1>
     <h3>Click on a country to start chatting!</h3>
     <div id="map"></div>
+    
```

Comment:
Hey, so feel free to make whatever changes you'd like to the writeup. I like to write a lot haha, but you don't have to use all of it!